### DISCUSSION TITLE
[Download] Custom Gamepad Layout

### DISCUSSION THREAD

**Discussion by zorazorr** [AppGround-io/bluetooth-keyboard-and-mouse-support#174] [Show and tell]
![GitHub Logo](https://raw.githubusercontent.com/zorazorr/bluetooth-keyboard-and-mouse-gamepad-layout/main/gamepad.png)

# [Download Gamepad Layout](https://github.com/zorazorr/bluetooth-keyboard-and-mouse-gamepad-layout/raw/main/gamepad.layout)

I know there is also an extra [gamepad app](https://play.google.com/store/apps/details?id=io.appground.gamepad), but unfortunately it doesn't work in my use case (mouse and gamepad needs to work with the same bluetooth-mac).

Here is a custom layout (workaround), that works with most emulators and game services.  
e.g.: Steam, RetroArch or RetroPie

Note / known issues:
- Touchpad can be used as thumbstick.
- Button combos aren't seamlessly and needs to be pressed button by button (e.g.: Street Fighter - Hadouken: ↓→ + Punch)

**Comment by SnickSCI** [comment]
Looks good! Thank you! I only needed to resize the buttons to my screen.

@AppGround-io 
Can you make it to switch between up, right, down, left without needing to take the thumb off the display?

And can you make analog sticks?
![1](https://play-lh.googleusercontent.com/bl0ywhGnYJ9bf5WoKmw9dcP8nwTVa-gfY64GgDyDdeYR53kDSfFCg3jMKORXmc_Kztg=w1824-h870)

![2](https://play-lh.googleusercontent.com/9J87jngEKXsy923l5VuA4_UNSjDNV0C3yBRtnyCbipgwt7s8v9-BZlbP-MDTH-z61w=w1824-h870)

![3](https://play-lh.googleusercontent.com/PBcUCCsAY7Jyq3w95dyMQ2U4mZL2QrO6H0m2u2oJNhLsN1Cj36Ftgj53im-Tq2A02eI=w1824-h870)
